execution_workflows/QAPA/README.md

```diff
@@ -1,11 +1,59 @@
-# APAeval nextflow pilot run
+# QAPA
+QAPA is a python-and-R-based model for quantifying alternative polyadenylation (APA) from RNA-seq data.
 
-### Steps to run this:
- - To run this first download the required reference files from [here](https://drive.google.com/drive/folders/1MUMilzaqef9u0sjScxzgi0JPTKAFQq_T?usp=sharing)
- - replace "path_to" in samplesheet_example_files.csv with the path to the reference files downloaded from the Google Drive link
- - check the path to this directory with `pwd` and replace the `[pwd]` in samplesheet_example_files.csv with the path from the `pwd` command
- - Before running `QAPA`, you have to install `QAPA`, and the guide to install is [here](https://github.com/morrislab/qapa). Make sure that QAPA is on your $PATH
- - Then, you are good to run the pilot benchmark nextflow pipeline with `QAPA`
+The [paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1414-4) is titled QAPA: a new method for the systematic analysis of alternative polyadenylation from RNA-seq data <br>
+The [application download](https://github.com/morrislab/qapa) is free to download
+and [documentation](https://github.com/morrislab/qapa#rna-seq-quantification-of-alternative-polyadenylation-qapa) was used as a reference
+to create the nextflow pipeline flow of this module
+
+## Running the QAPA workflow
+
+### Input & pre-processing
+An example sample sheet is available at `samplesheet_example.csv`. Each row in the samplesheet has nine
+columns:
+
+- sample: name of the sample for logs (e.g control_replicate1)
+- fastq1: FASTQ1 for both single-end and paired-end RNA-seq
+- fastq2: FASTQ2 for paired-end RNA-seq. If single-end, please leave this empty
+
+When using your own data and input file instead of the provided test data and sample sheet, make sure to include in the 
+input file you are using the absolute paths to the files, with the column names following the column
+names above.
+
+### Running with Docker or Singularity
+## Docker
+This workflow uses docker containers. To run with docker, make sure that docker is installed and running 
+(e.g. to ensure docker is running, run the command `docker --help` and a help message should be printed).
+To run with `docker`, please indicate `-profile docker`
+```
+nextflow main.nf --input samplesheet_example_files.csv` --gtf <path_to_gtf> --polyabed <path_to_poly(A)_bed> --fasta <path_to_fasta> --run_qapa_build -profile docker
+```
+
+## Singularity
+To run with `singularity`, please indicate `-profile singularity`
+```
+nextflow main.nf --input samplesheet_example_files.csv` --gtf <path_to_gtf> --polyabed <path_to_poly(A)_bed> --fasta <path_to_fasta> --run_qapa_build -profile singlularity
```

Comment:
Could this section be moved below the 'parameters' section and joined with the 'running the workflow' section? I think it would read a bit better

Reply:
I have moved this section below parameter and joined with the 'running the workflow' section